home/home.html

```diff
@@ -6,47 +6,74 @@
     <link rel="icon" href="..\EasyShopping2.0\assets/fast-trolly.png" type="image/png">
     <link rel="stylesheet" href="style.css">
     <link rel="icon" href="..\EasyShopping2.0\assets/fast-trolly.png"type="image/png">
-  <title>Easy Shopping home</title>
+    <title>Easy Shopping home</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <script type="text/javascript" src="app.js"></script>
 </head>
 
 <body>
-    <header>
-    <div>EasyShopping</div>
-    <nav>
-      <ul>
+  <header>
+    
+    <nav class="navbar navbar-expand-sm">
+      <div class="navbar-brand">EasyShopping</div>
+      <!-- <ul >
         <li><a href="home.html">Home</a></li>
-        <li><a href="../contact-us/contact-us.html">Contact Us</a></li>
-        <li><a href="..\shopper\shopper.html">Shopper</a></li>
-        <li><a href="..\writer\writer.html">Writer</a></li>
+        <li><a href="./contact-us/contact-us.html">Contact Us</a></li>
+        <li><a href=".\shopper\shopper.html">Shopper</a></li>
+        <li><a href=".\writer\writer.html">Writer</a></li>
 
-      </ul>
+      </ul> -->
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item active">
+            <a class="nav-link" href="home.html">Home <span class="sr-only">(current)</span></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="../contact-us/contact-us.html">contact</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="..\shopper\shopper.html">shopper</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="..\writer\writer.html">writer</a>
+          </li>
+        </ul>
```

Comment:
the hrefs don't work while running on the server. They only work while the server is offline mode and the file is run as an html file in the browser